src/main/java/frc/robot/Constants.java

```diff
@@ -18,7 +18,30 @@ public class Drive {
         public static final double WHEEL_DIAMETER_METERS = Units.inchesToMeters(4);
 
     }
-
+    public class Arm{
+        //Encoder constants
+        public static final int ENCODER_CHANNEL_A = 28;
+        public static final int ENCODER_CHANNEL_B = 29;
+        public static final int ENCODER_COUNT = 8192;
+        //Home switch constants
+        public static final int HOMESWITCH_CHANNEL = 30;
+        //Motor constants
+        public static final int ARM_MOTOR_ONE = 15;
+        public static final int ARM_MOTOR_TWO = 16;
```

Comment:
These should be labeled `ARM_MOTOR_LEFT` and `_RIGHT` with respect to the robot (facing the same direction as the robot, left is on the left). Work with electrical to determine which of 15 and 16 are which